src/content/en/tools/chrome-devtools/console/command-line-reference.md

```diff
@@ -2,8 +2,9 @@ project_path: /web/tools/_project.yaml
 book_path: /web/tools/_book.yaml
 description: A reference of convenience functions available in the Chrome DevTools Console.
 
-{# wf_updated_on: 2017-04-07 #}
+{# wf_updated_on: 2018-06-27 #}
 {# wf_published_on: 2015-04-13 #}
+{# wf_blink_components: Platform>DevTools #}
 
 # Command Line API Reference {: .page-title }
 
```

Comment:
Are you going to put a brief sentence into the top of this doc and the Console API doc explaining the difference between the two, as we discussed?